packages/web-api/src/WebClient.spec.js

```diff
@@ -149,7 +135,6 @@ describe('WebClient', function () {
             assert.equal(error.code, ErrorCode.RequestError);
             assert.equal(error.original.config.timeout, timeoutOverride);
             assert.equal(error.original.isAxiosError, true);
-            assert.equal(error.original.request.aborted, true);
```

Comment:
In axios 1.0, this assertion no longer passes. Not sure how important that is. Let me know if you have opinions on that.

Reply:
I don't see any issues with the deletion